.circleci/config.yml

```diff
@@ -134,7 +134,7 @@ jobs:
       - run: npm run build
       - run: PYTHONPATH=. python scripts/nightly/get_version.py > __init__.py
       - run: mv __init__.py wagtail/__init__.py
-      - run: python setup.py bdist_wheel
+      - run: python pyproject.toml bdist_wheel
```

Comment:
This does not work, you need to use the tool called `build`

Reply:
Mistakenly did replace all 🙈 thanks